index.js

```diff
@@ -2,34 +2,38 @@
 var path = require('path');
 var childProcess = require('child_process');
 
-module.exports = function (target, app, cb) {
+module.exports = function (target, opts, cb) {
 	if (typeof target !== 'string') {
 		throw new Error('Expected a `target`');
 	}
 
-	if (typeof app === 'function') {
-		cb = app;
-		app = null;
+	if (typeof opts === 'function') {
+		cb = opts;
+		opts = {};
+	} else if (!opts) {
+		opts = {};
 	}
 
+	if (opts.wait !== false) opts.wait = true
```

Comment:
Just use https://github.com/sindresorhus/object-assign so not to modify the user supplied options object ;)